pkg/nsx/services/subnet/store.go

```diff
@@ -39,6 +39,24 @@ func subnetIndexFunc(obj interface{}) ([]string, error) {
 	}
 }
 
+func subnetIndexNamespaceFunc(obj interface{}) ([]string, error) {
```

Comment:
Would this be better as both the only difference between 2 tags is VM instead of Scope?
subnetIndexNamespaceFunc -> subnetIndexVMNamespaceFunc // match TagScopeVMNamespace
subnetIndexScopeNamespaceFunc -> subnetIndexNamespaceFunc // match TagScopeNamespace

Reply:
Good catch.Updated.